learnr_bank/learnr_set2.Rmd

```diff
@@ -0,0 +1,468 @@
+  + **Hint:** `5:7` defines an array with values 5, 6, 7. Can you adapt this approach to the given question
+  + Solution: array_numbers2 <- 1:5
+              array_numbers2
+              
+``` {r array_q2, exercise = TRUE}
+
+```  
+
+<div id= "array_q2-hint">
+**Hint:**  `5:7` defines an array with values 5, 6, 7. Can you adapt this approach to the given question
+</div>
+
+3. Define an array variable called `even_numbers` that contains the values 2, 4, 6, 8, and 10 using the `c()` function.
+  + (no hint)
+  + Solution: even_numbers <- c(2,4,6,8,10)
+              even_numbers
+
+``` {r array_q3, exercise = TRUE}
+
+```  
+
+4. Define an array variable called `even_numbers` that contains the values 2, 4, 6, 8, and 10 using a new function (!) `seq()`. The chunk below shows an example of using `seq()`. Run this code to see the function in action, and then *adapt* the code to make the `even_numbers` array.
+
+  + **Hint 1**: You may want to run `?seq` to see more about how the function works!
+  + **Hint 2**: Always print out your output to be sure the variable you've created is correct. Keep trying until you get there - coding is all about trial and error! 
+  + Solution: even_numbers <- seq(2, 10, 2)
+              even_numbers
+
+```{r array_q4, exercise = TRUE}
+seq(1, 34, by = 3)
+
+```    
+
+```{r array_q4-hint-1}
+#You may want to run `?seq` to see more about how the function works!
+```
+
+```{r array_q4-hint-2}
+#Always print out your output to be sure the variable you've created is correct. Keep trying until you get there - coding is all about trial and error! 
+```
+
+5. The function `length()` tells you how many items are in a given array. Define an array using `seq()` as the multiples of 3 from 0-999 (i.e. 0, 3, 6, 9, ..., 993, 996, 999) and then use `length` to determine how many numbers it contains.
+  + **Hint**: First make sure (print print print!) that you are defining this array correctly. *Only once* you can see that the array is correct should you calculate its length.
+  + Solution: mul3 <- seq(0, 999, 3)
+              length(mul3)
+ 
+```{r array_q5, exercise = TRUE}
+
+```    
+
+<div id= "array_q5-hint">
+**Hint:** First make sure (print print print!) that you are defining this array correctly. *Only once* you can see that the array is correct should you calculate its length.
+</div>
+
+<!-- a lot of questions coming up should be pre-populated with baseline chunk code--> 
+6. One place where R really shines is *vectorization* - it can perform an operation on *all items in a vector* at the same time! The code below multiplies every value in the array `example_array` by 5. Run this code and examine its output to be sure you understand it. Then, modify the code to instead divide every item in the array by 10.
+  + Baseline chunk code: 
+        example_array <- c(5, 78, -3, 11, 6, 0)
```

Comment:
I noticed that in question 6, the only number in the array that was different was "1" and "11". I just wanted to double check if this is what you intended to do.

Reply:
Ah, my bad - any set of numbers here is fine, and you can fill in the solution dividing it by 10!